src/main/java/mathandel/backend/model/server/enums/RateName.java

```diff
@@ -0,0 +1,8 @@
+package mathandel.backend.model.server.enums;
+
+public enum RateName {
+    //todo nie pamietamjak sie miały nazywac xd plus mozna nadac im wartosci i potem srednia bedzie latwo liczyc
+    NAME1, NAME2, NAME3, NAME4;
```

Comment:
I dont think that values whould be good idea maybe we could just count the number of each and return it with endpoint getUserData,

Names:
NotReceived - transacion not ok
BadState - worse than in description (missing sth or just bad contdition)
AllGood - transaction ok